src/python/platforms/android/adb.py

```diff
@@ -604,7 +604,12 @@ def reset_usb():
 
   # We need to get latest device path since it could be changed in reboots or
   # adb root restarts.
-  device_path = get_device_path()
+  try:
+    device_path = get_device_path()
+  except IOError:
```

Comment:
This feels a little too generic, why not do the try-catch around line 580-581 and return None there ?

Reply:
I'm fine with that if you'd prefer it. The main reason to do it here was to ensure that we see errors if we fail when we set the value for DEVICE_PATH, which could cause more trouble later.

Reply:
This is fine.